resources/event-bus/values.yaml

```diff
@@ -2,10 +2,17 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 global:
+  containerRegistry:
+    path: eu.gcr.io/kyma-project
+  event_bus:
+    dir: develop/
+    version: ab9c73dd
+  event_bus_tests:
+    dir: develop/
+    version: f01c8848
   natsStreaming:
     url: "http://nats-streaming.natss:4222"
     clusterID: "kyma-nats-streaming"
```

Comment:
clusterID, as discussed, should be a global variable, cause for Knative deployment its value is: "knative-nats-streaming"

Reply:
@radufa Yes, this is coming in the next commit.